docs/src/AvatarPair.doc.js

```diff
@@ -28,6 +29,30 @@ card(
   />,
 );
 
+card(
+  <MainSection name="Usage guidelines">
+    <MainSection.Subsection columns={2}>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        title="When to Use"
+        description={`
+          - To represent a group of people, companies and/or brands in a square format.
+          - In cases where the parent element can represent either a single person/company or a group of people/companies.
```

Comment:
I'm not sure I totally understand this one.... What does parent element refer to?

Reply:
Yeah... I'm having a hard time communicating this one. Basically, there are cases where UI will represent a individual account or a group of accounts in the same location (essentially, Inbox). I'm fumbling over how to best communicate this.

Reply:
This sounds much better, love it!